.github/workflows/checks-quick.yml

```
@@ -35,6 +35,18 @@ jobs:
         with:
           app-id: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_ID }}
           app-key: ${{ secrets.WORKFLOW_STOPPER_RUNNER_APP_KEY }}
+  prdoc:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    needs: [preflight]
+    container:
+      image: ${{ needs.preflight.outputs.IMAGE }}
```

```suggestion
    container:
      image: docker.io/paritytech/prdoc:v0.1.1
```
We can use the prdoc docker image directly and remove the preflight dependency for this job

---

nice

---

The job fails with:
```
Run actions/checkout@6d193bf28034eafb982f37bd894289fe649468fc
  with:
    repository: paritytech/polkadot-sdk
    token: ***
    ssh-strict: true
    ssh-user: git
    persist-credentials: true
    clean: true
    sparse-checkout-cone-mode: true
    fetch-depth: 1
    fetch-tags: false
    show-progress: true
    lfs: false
    submodules: false
    set-safe-directory: true
/usr/bin/docker exec  77049c6e515a0f2aa2135b3aa7284a6103dffc1b04917d2f5e35669322f7a9e1 sh -c "cat /etc/*release | grep ^ID"
node:fs:2367
    return binding.writeFileUtf8(
                   ^

Error: EACCES: permission denied, open '/__w/_temp/_runner_file_commands/save_state_25d8542c-7fb7-41e3-be90-e0e7f532310e'
    at Object.writeFileSync (node:fs:2367:20)
    at Object.appendFileSync (node:fs:2448:6)
    at Object.issueFileCommand (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:3032:8)
    at Object.saveState (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:2949:31)
    at 4866 (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:2402:10)
    at __nccwpck_require__ (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:38166:43)
    at 2565 (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:150:34)
    at __nccwpck_require__ (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:38166:43)
    at 9210 (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:1171:36)
    at __nccwpck_require__ (/__w/_actions/actions/checkout/6d193bf28034eafb982f37bd894289fe649468fc/dist/index.js:38166:43) {
  errno: -13,
  code: 'EACCES',
  syscall: 'open',
  path: '/__w/_temp/_runner_file_commands/save_state_25d8542c-7fb7-41e3-be90-e0e7f532310e'
}
```

Maybe some permission issue?

---

oh, yeah, it uses non-root user inside of the container